src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -224,6 +230,26 @@ export const AdministrationPage = () => {
           </ServiceContainerCreatingUpdating>
         </FlexItem>
 
+        <FlexItem>
+          <ContentBox padding title="Git version">
```

Comment:
I would remove `title` attribute or I think "Web UI Git SHAs" would be probably more self-explaining